schemas/expconf/v0/experiment.json

```diff
@@ -108,6 +108,14 @@
                 "type": "string"
             }
         },
+        "log_pattern_policies": {
```

Comment:
There's something about the wording that isn't sitting well with me. Let's say a policy is "when you encounter this phenomenon (or are in this state), do this thing"? In that definition, this could just be a `log_policy`, and that policy could have a `pattern` and an `action`.

The wording/type for `pattern` makes sense to me, and further down in the file I've suggested an alternative to the type definition for `action`, but here I'd also like to suggest changing the enums to look more like actions. Namely:
```
on_failure_exclude_node -> exclude_node
on_failure_dont_retry -> cancel_retries
```
If the policy is being activated, the condition is already met.

I think this formulation also resolves ambiguity for what happens if a row matches multiple patterns. When they're policies, the reader wonders which policy is chosen to apply. If they're actions, well, the answer is all of them.

Reply:
Nice! I like how it looks!